tests/lib/ntn/Kconfig

```diff
@@ -0,0 +1,18 @@
+#
+# Copyright (c) 2025 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+config NTN
+	bool "Internal"
+	#depends on NRF_MODEM_LIB
```

Comment:
why is this commented out ?

Reply:
Answering here to both comments regarding the redefinition of `NTN`. These are done to remove the dependency to `NRF_MODEM_LIB` from the unit tests, as we don't want to enable it in the tests.

Reply:
To remove the dependency compared to the original definition. And if the question is that why it's commented out and not removed, I like when you see all lines of the original definition here in the unit tests so this documents nicely what has been removed for the purposes of the tests.